core/controllers/contributor_dashboard_admin.py

```diff
@@ -594,15 +595,17 @@ class ContributorDashboardAdminStatsHandler(
                 },
                 'default_value': None
             },
-            'max_days_since_last_activity': {
+            'start_date_locale_string': {
                 'schema': {
-                    'type': 'int',
-                    'validators': [{
-                        'id': 'is_at_least',
-                        'min_value': 0
-                    }]
+                    'type': 'basestring'
                 },
-                'default_value': None
+                'default_value': '1/1/3000'
```

Comment:
Same as above, please look at the most common date format used in the project.

Reply:
hello @nikitaevg,
I looked into the project and found out the in python files there are multiple date formats currently in use like `'%Y-%m-%d'` format, `'%m/%d/%Y, %H:%M:%S:%f'` format, `'%d %b %Y'` format, `'%b %d, %Y'` format.
In TS files `('en-us', {year: 'numeric',  month: 'long', day: 'numeric',})`, `('en-us', {year: 'numeric',  month: 'short', day: '2-digit',})` formats are used.
could you please help me in figuring out which format should I use.

Reply:
I am thinking of going with `'%Y-%m-%d'` format as it is used most of the time

Reply:
> '%Y-%m-%d'

This one is the best IMO and it's widely, so yeah, go with it

Reply:
done